tests/e2e/server.php

```diff
@@ -21,7 +21,8 @@
 App::get('/chunked')
     ->inject('response')
     ->action(function ($response) {
-        /** @var Utopia/Response $response */
+        /** @var Utopia /Response $response
+         * @phpstan-ignore-next-line */
```

Comment:
This should probably be:

```suggestion
    ->action(function (Response $response) {
```

Reply:
Thanks, will apply the suggested change.